scripts/es-integration-test.sh

```diff
@@ -10,23 +10,19 @@ usage() {
   echo $"Usage: $0 <elasticsearch|opensearch> <version>"
   exit 1
 }
-
+#check if the number of arguments is correct
 check_arg() {
   if [ ! $# -eq 3 ]; then
-    echo "ERROR: need exactly two arguments, <elasticsearch|opensearch> <image> <jaeger-version>"
+    echo "ERROR: need exactly three arguments, <elasticsearch|opensearch> <image> <jaeger-version>"
     usage
   fi
 }
-
+#start the elasticsearch/opensearch container
 setup_db() {
-  local distro=$1
-  local compose_file=$2
+  local compose_file=$1
   docker compose -f "${compose_file}" up -d
-  local cid 
-  cid=$(docker compose -f "${compose_file}" ps -q "${distro}")
-  echo "cid=${cid}" >> "$GITHUB_OUTPUT"
```

Comment:
can we keep this?
```
echo "docker_compose_file=${compose_file}" >> "$GITHUB_OUTPUT"
```
then use it in the workflow instead of hardcoding the file path

Reply:
Actually even better: `>> "${GITHUB_OUTPUT:-/dev/null}"`

this way when the script is run manually / locally it will not fail on missing GITHUB_OUTPUT env var